gpkit/geometric_program.py

```diff
@@ -415,9 +415,10 @@ def _solve_sweep(self, printing, skipfailures, allownonoptimal):
         self.presweep = self.last
 
         if len(self.sweep) == 1:
-            sweep_grids = np.array(self.sweep.values())
+            sweep_grids = np.array(list(self.sweep.values()))
```

Comment:
in py3, `.values()` returns a `dictvalues` object, which is iterable but not a list.